execute/exectypes/costly_messages_test.go

```diff
@@ -2,33 +2,38 @@ package exectypes
 
 import (
 	"context"
+	"fmt"
 	"math/big"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
 
 	"github.com/smartcontractkit/chainlink-ccip/internal/plugintypes"
+	gasmock "github.com/smartcontractkit/chainlink-ccip/mocks/execute/internal_/gas"
 	readerpkg_mock "github.com/smartcontractkit/chainlink-ccip/mocks/pkg/reader"
 	"github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
 )
 
-func TestCCIPCostlyMessageObserver_Observe(t *testing.T) {
-	b1, err := ccipocr3.NewBytes32FromString("0x01")
-	if err != nil {
-		t.Fatal(err)
-	}
-	b2, err := ccipocr3.NewBytes32FromString("0x02")
-	if err != nil {
-		t.Fatal(err)
-	}
-	b3, err := ccipocr3.NewBytes32FromString("0x03")
+var (
+	b1 = bytesOrPanic("0x01")
+	b2 = bytesOrPanic("0x02")
+	b3 = bytesOrPanic("0x03")
+)
+
+func bytesOrPanic(s string) ccipocr3.Bytes32 {
```

Comment:
nit: ["must" is the conventional name](https://google.github.io/styleguide/go/decisions.html#must-functions)
```suggestion
func mustMakeBytes(s string) ccipocr3.Bytes32 {
```